pvlib/modelchain.py

```diff
@@ -447,7 +447,7 @@ def with_pvwatts(cls, system, location,
 
         >>> pvwatts_losses = {'soiling': 2, 'shading': 3, 'snow': 0, 'mismatch': 2,
         >>>                   'wiring': 2, 'connections': 0.5, 'lid': 1.5,
-        >>>                   'nameplate_rating': 1, 'age': 0, 'availability': 30}
+        >>>                   'nameplate_rating': 1, 'age': 0, 'availability': 3}
```

Comment:
I see how 30 might have meant 3, but now these add up to 15% rather than the 14% stated in the docstring, so should this be 2 rather than 3, or is something else (the docstring?) wrong here?
@cwhanse @kandersolar ?
```suggestion
        >>>                   'nameplate_rating': 1, 'age': 0, 'availability': 2}
```

Reply:
Thanks for clarifying, that's my mistake! You can mark this conversation as resolved, and then one of the other maintainers will approve workflows and double check the PR before merging.

Reply:
Indeed PVWatts losses combine multiplicatively; see the source code of `pvlib.pvsystem.pvwatts_losses`.

Regarding the typo: from the comment immediately above the dict, it looks like those parameter values are not supposed to reflect the defaults but rather an example set of user-specified values.  I think the 30 was intentional then, and it defeats the purpose of the example to change it back to the default value :)

Reply:
> * revert the typo that wasn't a typo, commit locally and then push to my repository

Sounds correct to me:)
You can make the change manually in the file, or I think there is a function in git to revert a specific commit too. Either is fine.